test/functional/feature_llmq_signing.py

```diff
@@ -119,7 +119,7 @@ def assert_sigs_nochange(hasrecsigs, isconflicting1, isconflicting2, timeout):
         assert(node.quorum("verify", 100, id, msgHash, recsig["sig"]))
         assert(node.quorum("verify", 100, id, msgHash, recsig["sig"], "", height))
         assert(not node.quorum("verify", 100, id, msgHashConflict, recsig["sig"]))
-        assert_raises_rpc_error(-8, "quorum not found", node.quorum, "verify", 100, id, msgHash, recsig["sig"], "", height_bad)
+        assert not node.quorum("verify", 100, id, msgHash, recsig["sig"], "", height_bad)
```

Comment:
let's not mix formatting like this?

Reply:
Thats the format i just always use since you had an issue with `assert()` a while ago and because this file anyway has mixed formats already i would rather stay format consistent within this PR.